nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-ui/src/main/webapp/js/nf/canvas/nf-settings.js

```diff
@@ -1712,14 +2582,17 @@
                     name: 'General',
                     tabContentId: 'general-settings-tab-content'
                 }, {
-                    name: 'Reporting Task Controller Services',
+                    name: 'Controller Level Controller Services',
```

Comment:
The user guide is still referencing `Reporting Task Controller Services`.

Reply:
This will be addressed in a separate Jira, which will include new screenshots too.

Reply:
Here is the Jira: https://issues.apache.org/jira/browse/NIFI-10501